### ISSUE TITLE
"Stamp" zones in the layout 

### ISSUE THREAD

**Issue by benfavre** [haltu/muuri#395]
I'm having a hard time understanding how I would implement "no go zones" aka "stamps" (name used by packery).
Do I need to do this by implementing a custom layout algorithm.

E.g.: https://codepen.io/agence-web-brest-webdesign29/pen/OJMgbev (my stamps are yellow)

**Comment by niklasramo** [comment]
Yep, custom layout algorithm would be the way. This is not the first time this is asked/requested though and I've been thinking about adding support for stamps (in some manner) to the default layout algorithm too, but never have gotten really into making it happen.
